src/Button/_mixins.scss

```diff
@@ -16,7 +16,29 @@
   @include border-radius($button-border-radius, 0);
 }
 
-// Mixin for button states that affect border radius
+/// Sets the CSS custom properties for a button variant
+/// @param {String} $variant - The variant name (e.g., 'primary', 'outline-primary')
+@mixin button-variant($variant) {
```

Comment:
[inform] Makes the setting of these CSS variables for each `.btn-<variant>` class name definition more DRY/reusable.

Reply:
This is great! If you want to break this out into a tiny separate PR I'd be happy to land this change standalone while we work through the details of the rest of this PR.